src/zeek-setup.cc

```diff
@@ -549,6 +549,7 @@ SetupResult setup(int argc, char** argv, Options* zopts)
 		add_input_file("base/init-default.zeek");
 
 	add_input_file("builtin-plugins/__load__.zeek");
+	add_input_file("builtin-plugins/__preload__.zeek");
```

Comment:
this is unrelated and part of https://github.com/zeek/zeek/pull/1615/files, right? If yes - it should probably not be part of this PR.

Reply:
Whoops, right.  That was in there because the work I was doing with this needed both fixes but I accidentally let this slip in.